src/java/main/org/apache/zookeeper/Version.java

```diff
@@ -20,7 +20,7 @@
 
 public class Version implements org.apache.zookeeper.version.Info {
 
-    public static int getRevision() {
+    public static String getRevision() {
```

Comment:
Good question! Um... Maybe we should contact folks at Kafka, HBase and other projects for feedback? Anyone has a clue?